docs/pages/machine-workload-identity/mwi-use-cases/mwi-ci-cd.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: CI/CD with Machine & Workload Identity
+description: Configuring CI/CD with Machine & Workload Identity
+labels:
+ - how-to
```

Comment:
I think this would make more sense as `conceptual`, since it's not a sequence of steps.